modules/admin-ui/src/main/webapp/scripts/shared/services/hotkeysService.js

```diff
@@ -52,31 +52,27 @@ angular.module('adminNg.services')
         });
       };
 
-      this.activateHotkey = function (scope, keyIdentifier, description, callback) {
+      function activateHotkey(hotkeys, keyIdentifier, callback) {
         me.loading.then(function () {
           var key = me.keyBindings[keyIdentifier];
-          if (key !== undefined) {
-            hotkeys.bindTo(scope).add({
-              combo: key,
-              description: description,
-              callback: callback
-            });
-          }
-        });
-      }
-
-      this.activateUniversalHotkey = function (keyIdentifier, description, callback) {
-        me.loading.then(function () {
-          var key = me.keyBindings[keyIdentifier];
-          if (key !== undefined) {
+          if (key) {
             hotkeys.add({
               combo: key,
-              description: description,
+              description: keyIdentifier,
```

Comment:
That reads somewhat weird... if keyIdentifier now an identifier or a description?

Reply:
Weeell, from our point of view it is an identifier, which we use to look up translations and also arrange for the grouping. For `angular-hotkeys` it is a description, but having overridden the default cheat sheet template that interpretation is meaningless. `angular-hotkeys` only uses the description in it's template. It is however the only place to put custom information to identify a hotkey later on in our own template and controller.

So yes, it looks strange, but calling the `HotkeyService` parameter `description` would be a lie, too, and unfortunately I can't (reasonably) change the parameter name that `angular-hotkeys` uses. :unamused:

Reply:
I could add a comment, if that helps. :slightly_smiling_face: